MEDIASOUP_PROTOCOL.md

```diff
@@ -336,6 +336,90 @@ Notification:
 ```
 
 
+### enableConsumerStats
+
+Enable periodic consumer stats retrieval.
+
+Notification:
+
+```js
+{
+  method: 'enableConsumerStats',
+  id: 3333,
+}
+```
+
+
+### disableConsumerStats
+
+Disable periodic consumer stats retrieval.
+
+Notification:
+
+```js
+{
+  method: 'disableConsumerStats',
+  id: 3333,
+}
+```
+
+
+### enableProducerStats
+
+Enable periodic producer stats retrieval.
+
+Notification:
+
+```js
+{
+  method: 'enableProducerStats',
+  id: 3333,
+}
+```
+
+
+### disableProducerStats
+
+Disable periodic producer stats retrieval.
+
+Notification:
+
+```js
+{
+  method: 'disableProducerStats',
+  id: 3333,
+}
+```
+
+
+### enableTransportStats
+
+Enable periodic transport stats retrieval.
+
+Notification:
+
+```js
+{
+  method: 'enableTransportStats',
+  id: 3333,
+}
+```
+
+
+### disableTransportStats
+
+Disable periodic transport stats retrieval.
+
+Notification:
+
+```js
+{
+  method: 'disableTransportStats',
+  id: 3333,
+}
+```
+
+
```

Comment:
Please, group API messages by entity (Transport, Producer, Consumer), possibly using the same order as `Room.js` handles them.